CHANGELOG

```diff
@@ -19,6 +19,14 @@
   they stand out in the output.
   Thanks Gabriel Reis for the PR.
 
+- add more talks to the documentation
```

Comment:
Not sure if this is worth a changelog entry (and the next one too), but your decision :wink: